### PR TITLE
Supplier desing.development

### PR DESCRIPTION
**Pull request by Dharmiik-Chauhan**
supplier changes done

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,24 @@
 module ApplicationHelper
+  def activate_if_current_page(path)
+    "active" if current_page?(path)
```
Comment (ManojParmar-MTAJ) [inline app/helpers/application_helper.rb]: create constant of 'active' string and use

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+  end
+
+  def devise_error_messages!
+    byebug
+    return "" unless devise_error_messages?
```
Comment (ManojParmar-MTAJ) [inline app/helpers/application_helper.rb]: just write return no need to write ""

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,24 @@
 module ApplicationHelper
+  def activate_if_current_page(path)
+    "active" if current_page?(path)
+  end
+
+  def devise_error_messages!
```
Comment (ManojParmar-MTAJ) [inline app/helpers/application_helper.rb]: remove this method from here we have added same code in devise helper.

File: app/helpers/devise_helper.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+module DeviseHelper
+  def devise_error_messages!
+    return "" unless devise_error_messages?
```
Comment (ManojParmar-MTAJ) [inline app/helpers/devise_helper.rb]: remove "" string just write return

File: app/helpers/devise_helper.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+    messages = resource.errors.full_messages.map { |msg| content_tag(:li, msg) }.join
```
Comment (ManojParmar-MTAJ) [inline app/helpers/devise_helper.rb]: use collect instead of map

File: app/views/devise/confirmations/new.html.erb [hunk 6]
```diff
@@ -1,19 +1,20 @@
...
 
-  <div class="actions">
-    <%= f.submit "Resend confirmation instructions"  class: "btn btn-primary" %>
-  </div>
-  <% end %>
\ No newline at end of file
+    <div class="actions">
+      <%= f.submit "Resend confirmation instructions" class: "btn btn-primary" %>
+    </div>
+  <% end %>
```
Comment (ManojParmar-MTAJ) [inline app/views/devise/confirmations/new.html.erb:20]: fix indentation of whole file

File: app/views/devise/passwords/new.html.erb [hunk 7]
```diff
@@ -1,20 +1,31 @@
...
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Send me reset password instructions",class: "btn btn-primary" %>
+    </div>
   <% end %>
 
-  <%= render "devise/shared/links" %>
\ No newline at end of file
+  <%= render "devise/shared/links" %>
```
Comment (ManojParmar-MTAJ) [inline app/views/devise/passwords/new.html.erb:31]: fix indentation of whole file

File: app/helpers/devise_helper.rb (hunk 5 above)
Comment (Dharmiik-Chauhan) [reply app/helpers/devise_helper.rb]: done

File: app/helpers/devise_helper.rb (hunk 4 above)
Comment (Dharmiik-Chauhan) [reply app/helpers/devise_helper.rb]: done

**Pull request merged**